srcpkgs/python3-mpi4py/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-mpi4py'
+pkgname=python3-mpi4py
+version=3.0.2
+revision=1
+wrksrc="mpi4py-${version}"
+build_style=python3-module
+pycompile_module="mpi4py"
+hostmakedepends="python3-setuptools python3-Cython gcc-fortran openmpi"
+makedepends="python3-devel openmpi-devel"
+depends="openmpi"
```

Comment:
Thanks for your first contribution to Void! Unless the package needs the openmpi binaries at runtime, please remove the `depends` line. If packages depend on shared libraries, these dependencies are automatically detected by `xbps-src` while building the package. The build process automatically adds a dependency to `libopenmpi>=3.0.1_1` in this case.

Reply:
Thanks Helmut! It does need openmpi at runtime. Python needs to be called with `/usr/bin/mpiexec` for mpi4py to take effect.